src/widgets/numeric-menu/__tests__/numeric-menu-test.js

```diff
@@ -103,7 +106,7 @@ describe('numericMenu()', () => {
 
   it("doesn't call the refinement functions if not refined", () => {
     widget.render({ state, results, createURL });
-    expect(helper.state.clearRefinements).toHaveBeenCalledTimes(
+    expect(helper.state.removeNumericRefinement).toHaveBeenCalledTimes(
       0,
       'clearRefinements called one'
```

Comment:
we should update all theses messages now

Reply:
https://github.com/algolia/instantsearch.js/pull/4106